apps/nova/src/components/TutorProfile/TutorTabs.tsx

```diff
@@ -7,6 +7,7 @@ import { LocalId } from "@litespace/luna/locales";
 import cn from "classnames";
 import { AnimatePresence, motion } from "framer-motion";
 import ProfileInfo from "./ProfileInfo";
+import Ratings from "./Ratings";
```

Comment:
Remember to always use absolute paths. You can always spot this by maing a self-review once you opend the pull request.